cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -327,6 +327,7 @@ public static KafkaCluster fromCrd(Reconciliation reconciliation, Kafka kafkaAss
             result.clusterId = getOrGenerateKRaftClusterId(kafkaAssembly);
         }
 
+        ModelUtils.validateComputeResources(kafkaClusterSpec.getResources(), ".spec.kafka.resources");
```

Comment:
I was wondering if we want to pass the `kafkaClusterSpec` here as for the `validateIntConfigProperty`. I also like the parameters order (first spec then what to check). Maybe in a different PR we could change such order for the `validateIntConfigProperty` as well.

Reply:
Thanks Paolo. 

> I was wondering if we want to pass the kafkaClusterSpec here as for the validateIntConfigProperty.

Here I'm passing the nearest spec object that contains the values to be validated. In that sense, I think it is consistent with `validateIntConfigProperty`.

> I also like the parameters order (first spec then what to check). Maybe in a different PR we could change such order for the validateIntConfigProperty as well.

Agree.